evaluate_manual_model.py

```diff
@@ -126,6 +137,13 @@ def compute_metrics(eval_predictions):
     model = MultipleChoiceModel()
     model.load_state_dict(torch.load(f'{model_mode}_trained_model/{model_mode}_trained_model.pt'))
     model.to(main_device)
+elif model_mode == "roberta":
+    import_model_name = "rinna/japanese-roberta-base"
+    tokenizer = T5Tokenizer.from_pretrained(import_model_name)
```

Comment:
Why T5 tokenizer?

Reply:
This is a mistake. I changed it to AutoTokenizer and committed.